tests/integration-tests/tests/tests-test.js

```diff
@@ -6,7 +6,9 @@ JSCK.Draft4 = JSCK.draft4;
 let artilleryCheck = new JSCK.Draft4(require('artillery/core/lib/schemas/artillery_test_script'));
 const requestSender = require('./helpers/requestCreator');
 const paymentsOsDsl = require('../../testExamples/paymentsos-dsl');
+const fileUrl = 'https://sample-videos.com/text/Sample-text-file-10kb.txt';
```

Comment:
maybe it would be better to use a file in our github?
for example:
https://raw.githubusercontent.com/Zooz/predator/master/README.md 

we know the readme will always be there

Reply:
Fixed